test/storage/offline_database.test.cpp

```diff
@@ -50,7 +45,7 @@ static FixtureLog::Message error(ResultCode code, const char* message) {
     return {EventSeverity::Error, Event::Database, static_cast<int64_t>(code), message};
 }
 
-static MBGL_UNUSED FixtureLog::Message warning(ResultCode code, const char* message) {
```

Comment:
Well, it seems `[[maybe_unused]]` cannot be used to suppress unused warnings for functions 🤷🏻

Reply:
Which compiler complains?

Reply:
Ah

```
/Users/ec2-user/actions-runner/_work/maplibre-native/maplibre-native/test/storage/offline_database.test.cpp:47:10: error: 'maybe_unused' attribute cannot be applied to types
static [[maybe_unused]] FixtureLog::Message warning(ResultCode code, const char* message) {
```